src/iframe/src/Iframe.js

```diff
@@ -574,7 +598,7 @@ class Iframe extends Component {
         })
 
         // Draw this state.
-        window.draw && window.draw(this.store.getState())
+        this.drawUserGraphics(this.store.getState());
```

Comment:
semicolons! i'd recommend you run prettier on everything, then you can totally forget about it

Reply:
Finally got it running. I run windows and prettier has some extra dependencies on windows.